public/index.html

```diff
@@ -0,0 +1,373 @@
+            </li>
+            <li class="design-list">
+                <figure data-tags="ui-design, brand">
+                <img src="./imgs/management.jpg" alt="screen displaying claim management site">
+                <figcaption><h4 class="design">Management</h4>
+                    <p>UI design, brand &#38; logotypes</p></figcaption>
+                </figure>
+            </li>
+            <li class="design-list">
+                <figure data-tags="web-design, brand">
+                <img src="./imgs/koreanbeef.jpg" alt="screen displaying korean beef recipe">
+                <figcaption><h4 class="design">Korean beef</h4>
+                    <p>Webdesign, brand &#38; logotypes</p></figcaption>
+                </figure>
+            </li>
+            <li class="design-list">
+                <figure data-tags="icon-design, brand">
+                <img src="./imgs/thumbs.jpg" alt="screen displaying thumbs up and down icons">
+                <figcaption><h4 class="design">Facebook thumbs</h4>
+                    <p>Icons design, brand &#38; logotypes</p></figcaption>
+                </figure>
+            </li>
+        </ul>
+    </section>
+    <section class="about-section">
+        <h1 id="about">ABOUT</h1>
+        <div class="column-two">
+            <p class="column-left">Lorem Ipsum is simply dummy text of the printing and type setting industry. Lorem Ipsum has been the industry standard dummy text since the 1500's, when an unknown printer took a galley of type and scrambled it to make a type specimen book</p>
+            <p class="column-right">Lorem Ipsum is simply dummy text of the printing and type setting industry. Lorem Ipsum has been the industry standard dummy text since the 1500's, when an unknown printer took a galley of type and scrambled it to make a type specimen book</p>
+        </div>
+        <h2 class="subheader subheader-skills">SKILLS</h2>
+        <div class="progress-diagram">
+            <figure>
+                <figcaption class="skill-list">PHOTOSHOP</figcaption>
+                <progress value="78" max="100"></progress>
+            </figure>
+            <figure>
+                <figcaption class="skill-list">HTML/CSS</figcaption>
+                <progress value="91" max="100"></progress>
+            </figure>
+            <figure>
+                <figcaption class="skill-list">ILLUSTRATOR</figcaption>
+                <progress value="63" max="100"></progress>
+            </figure>
+            <figure>
+                <figcaption class="skill-list">PHP</figcaption>
+                <progress value="55" max="100"></progress>
+            </figure>
+            <figure>
+                <figcaption class="skill-list">WORDPRESS</figcaption>
+                <progress value="97" max="100"></progress>
+            </figure>
+        </div>
+        <p class="skills-para">Do you want be one of Avengers? Join us!</p>
+        <button class="contact-link">CONTACT US</button>
```

Comment:
Use `<button>` for _actions_, `<a>` for _navigating_. A good rule of thumb is if it takes you somewhere, either on the page or elsewhere on the site, use an anchor; if it submits a form or performs some "action", use a button. This case is more the former, there's no action here.